### DISCUSSION TITLE
Release V1.0.11

### DISCUSSION THREAD

**Discussion by moabu** [JanssenProject/jans#4280] [Polls]
https://github.com/JanssenProject/jans/milestone/14

**Comment by yuriyz** [comment]
We can't release until full rebuild is fixed.

**Reply by manojs1978** [reply]
https://github.com/JanssenProject/jans/issues/4278
https://github.com/JanssenProject/jans/issues/4289

**Comment by christian-hawk** [comment]
> "Should we release V1.0.11?"

@moabu I don't get it.  Is this question regarding a deadline? if we don't release we gonna skip this version?

**Reply by moabu** [reply]
No . It's to validate we are ready for release. If a team member has an objection or a blocker not seen by any team leader or PM this would clarify that. In the event the poll shows a negative result it's a good indicator we should push the date of the release but not skip it .

**Comment by ossdhaval** [comment]
@moabu Two points to consider before release:
- Issue #4289 is related to Jans on SUSE. If SUSE is the primary distribution we are targeting, we should get this issue fixed. It is marked Priority-1. I think @devrimyatar can give an ETA once he analysis it. 
- The build that we release, should be tested OK by the QA team. We need to check with them how much time @manojs1978 needs to certify. This also means that we can't release from the tip of the `main` branch because we need to give a release candidate to the QA team which they can certify.

**Reply by manojs1978** [reply]
once we get release candidate ,min 2 days required for sanity testing.

**Reply by moabu** [reply]
The process is to put a code pause on main and keep the features in their respective feature branches/PRs. There is no hurry into pushing code into main unless its addressing a release blocker. A pause of one day makes sense. @manojs1978 two days is too much . 1 working day sounds more reasonable to run sanity tests.

**Reply by manojs1978** [reply]
we have to do sanity testing on all 4 OS and 3 db each. total 12 combinations.
we can try .

**Reply by yuriyz** [reply]
Actually this time we should test AS more because of changes in core of AS. Especially ACR change functionality. E.g. user is authenticated with acr1 and then authenticate with acr2. It's related to https://github.com/JanssenProject/jans/issues/4255

We should test at least these scenarios before release:
- device_authorization(TUI) -> basic
- device_authorization(TUI) -> simple_password_auth
- simple_password_auth -> basic
- basic -> simple_password_auth

**Reply by MohitMaliFtechiz** [reply]
@yuriyz i've checked acr change functionality with jans 1.0.11 nightly build (latest) installed on Ubuntu 22 and 20 which is working. i tried this scenario with  ``` simple_password_auth ``` and ``` otp ``` .

**Reply by yuriyz** [reply]
@MohitMaliFtechiz nice, thanks for testing.

**Comment by MohitMaliFtechiz** [comment]
Hello @all jans installation on rhel 8 is failed with all backends 
#4313 
#4312 
#4314
